server/testutil/testenv/testenv.go

```diff
@@ -104,11 +102,7 @@ func (te *TestEnv) GRPCServer(lis net.Listener) (*grpc.Server, func()) {
 }
 
 func GetTestEnv(t testing.TB) *TestEnv {
-	configureLoggerOnce.Do(func() {
-		*log.LogLevel = "debug"
-		*log.IncludeShortFileName = true
-		log.Configure()
-	})
+	testlog.Configure()
```

Comment:
Tyler might kill me for suggesting this, but what if we put this in a `init` func in testenv instead? 
Would avoid having to introduce this separate testlog utility.

Reply:
Now that I think about it, it actually used to be in an init() func -- Tyler was even the one who put it there IIRC :smile: 

@tempoz it looks like this was moved this to a `sync.Once` intentionally https://github.com/buildbuddy-io/buildbuddy/commit/3f08babfe01fa55b43c84477b1a6f564aa0fcd70 any chance you remember the context for that?

Reply:
I think this might be vestigial from the interstitial period when we still had to parse and reconcile flags; very possibly it would be fine to revert to the init function. I am 90% sure that the change was motivated by very obvious failures, so it should be pretty clear whether or not it's okay to change it.

Reply:
Thanks for looking at this - changed it back to an `init()` for now